libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/thrift/ThriftValidate.java

```diff
@@ -269,6 +270,9 @@ private static void validateLicenseIdMatch(License license) throws SW360Exceptio
     public static void prepareProject(Project project) throws SW360Exception {
         assertNotEmpty(project.getName());
         project.setType(TYPE_PROJECT);
+        if(!project.isSetClearingState()) {
+           project.clearingState =  ProjectClearingState.OPEN;
```

Comment:
is there a `project.setClearingState(ProjectClearingState.OPEN);` ?

Reply:
Whitespace and formatting problems:
```suggestion
            project.clearingState = ProjectClearingState.OPEN;
```

Reply:
Thanks for pointing it out. Made the necessary changes.